generator/src/main/java/com/scottlogic/deg/generator/restrictions/NumericRestrictionsMerger.java

```diff
@@ -21,13 +21,31 @@
 
         final NumericRestrictions merged = new NumericRestrictions();
 
-
         merged.min = getMergedLimitStructure(MergeLimit.MIN, left.min, right.min);
         merged.max = getMergedLimitStructure(MergeLimit.MAX, left.max, right.max);
 
+        if (!canEmitSomeNumericValues(merged)){
+            return new MergeResult<>(); //successful = false
+        }
+
         return new MergeResult<>(merged);
     }
 
+    private boolean canEmitSomeNumericValues(NumericRestrictions merged) {
+        NumericLimit<BigDecimal> min = merged.min;
+        NumericLimit<BigDecimal> max = merged.max;
+
+        if (min == null || max == null){
+            return true; //no constraints
+        }
+
+        if (min.isInclusive() && max.isInclusive()){
```

Comment:
Scenarios to consider are:

| min | max | outcome |
| ---- | ---- | ---- |
| > 5 | < 10 | tbc |
| >= 5 | < 10 | tbc |
| >= 5 | <= 10 | tbc |
| > 5 | <= 10 | tbc |
| > 5 | < 5 | tbc |
| > 5 | <= 5 | tbc |
| >= 5 | < 5 | tbc |
| >= 5 | <= 5 | tbc |
| >= 5 | < 3 | tbc |
| > 5 | < 3 | tbc |
| > 5 | <= 3 | tbc |
| >= 5 | <= 3 | tbc |